ooniprobe/Utility/MessageUtility.m

```diff
@@ -13,6 +13,36 @@ + (void)showToast:(NSString*)msg inView:(UIView*)view
     });
 }
 
++ (void)alertVpnWithTitle:(NSString *)title message:(NSString *)msg
+                   inView:(UIViewController *)view
+             runVPNAction: (void (^ __nullable)(UIAlertAction *action))runVPNAction
+        runAlwaysVPNAction: (void (^ __nullable)(UIAlertAction *action))runAlwaysVPNAction
+             disableVPNAction: (void (^ __nullable)(UIAlertAction *action))disableVPNAction
```

Comment:
Why are these functions declared as `__nullable`? Is there a case when they are `null`? Can we otherwise declare them as `__nonnull`?